### DISCUSSION TITLE
Query about sending an email after adding a submission

### DISCUSSION THREAD

**Discussion by benjamincaha** [globaleaks/globaleaks-whistleblowing-software#3963] [Q&A]
Hello,

is there any way to set the system to email me after each submission that something has been added? I have the notifications set to hours, but that's not very functional.

I would like to send an email after submitting that something has been added.  Is this somehow possible? I've gone through the settings but it's not possible.

Thanks.

**Comment by evilaliv3** [comment]
Thank you. I'm not sure if i understand the question.

The system already send an email as soon that something is added on a report.

**Reply by benjamincaha** [reply]
Hello,

I've discovered that I have the wrong SMTP server set up. But my provider forces me to put special characters @, # and at least one more in my password. Can't you check? Whether GlobalLeaks counts with a password where special characters can be entered?

Thanks.
